include/miopen/miopen.h

```diff
@@ -6582,6 +6582,78 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+
+/*! @ingroup Cumulative Reduction
```

Comment:
You can probably move `@addtogroup` up here instead of including it separately

Reply:
Fixed it